rustplus/utils/rust_utils.py

```diff
@@ -3,6 +3,7 @@
 from PIL import Image, ImageDraw, ImageFont
 import logging
 import string
+import requests
```

Comment:
Where did this come from

Reply:
Forgot to remove it, it's been around since I made the Steam Web API interlaced with the user avatars